projects/xkbcommon.org/package.yml

```diff
@@ -9,6 +9,10 @@ dependencies:
   x.org/xcb: '*'
   freedesktop.org/XKeyboardConfig: '*'
   gnome.org/libxml2: '*'
+runtime:
+  env:
+    XDG_CONFIG_HOME: "{{deps.freedesktop.org/XKeyboardConfig.prefix}}/share/X11"
+    XKB_CONFIG_EXTRA_PATH: "{{deps.freedesktop.org/XKeyboardConfig.prefix}}/share/locale"
```

Comment:
Isn't is likely that these should be defined on XKeyboardConfig?

Reply:
sure 👍🏻

Reply:
https://github.com/teaxyz/pantry/pull/3430